### ISSUE TITLE
When job is running, selected EE is not visible in the UI, it appears only after it finishes

### ISSUE THREAD

**Issue by benapetr** [ansible/ansible-ui#3085]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Feature type

New Feature

### Feature Summary

Right now EE that was selected is invisible to user, see here

![image](https://github.com/ansible/awx/assets/1560121/a61b4a40-d6ee-421f-932c-925716b0f996)

A specific version of EE was used for this template, but it's not visible anywhere in the job history (it was changed between runs) and therefore we can't verify if that specific version was really used or not and which version of EE the job was started with

### Select the relevant components

- [X] UI
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Steps to reproduce

Create a template, pick a custom EE, start a job, then edit template, pick another EE, start a new job. You won't be able to tell which job was started with which EE.

### Current results

See above

### Sugested feature result

Make it so that chosen EE version is visible just like other fields, job template, project etc.

**Comment by benapetr** [comment]
EE can be chosen here

![image](https://github.com/ansible/ansible-ui/assets/1560121/35ef91dd-e48c-41d2-ba22-57652f51673a)

But it's not possible to see which one was chosen for a job that is running (I just noticed this information appears later when job already finishes, but it should probably be visible even when it's running)

**Comment by kurokobo** [comment]
@benapetr 
Hi, indeed EE is not shown during the job run, but once the job is completed, the `Details` is updated to include used EE.

![image](https://github.com/ansible/ansible-ui/assets/2920259/39e4ac46-1f6e-442a-9e31-3ee9c38f0a13)

Does this not satisfy your requirement?

**Comment by marek1712** [comment]
@thedoubl3j it's shown for both Running and Completed jobs:
![image](https://github.com/ansible/ansible-ui/assets/107187302/78698399-c088-4845-9832-9138f5d578a8)
I guess that's matter of some condition when Details page is generated. This?
https://github.com/ansible/awx/blob/d35d8b6ed7511912f1b3c05aa64ec7302680c5e8/awx/ui/src/screens/Job/JobDetail/JobDetail.js#L335-L340

**Comment by keithjgrant** [comment]
This codebase has been significantly reworked, and we are closing old issues that predate these changes. If the issue still persists in the new `devel` branch, please file a new issue.
